src/graph/connect.cc

```diff
@@ -74,7 +75,7 @@ ncclResult_t ncclTopoPreset(struct ncclComm* comm, struct ncclTopoGraph** graphs
   // Get nvls heads and the number of heads. Duplicate head is not allowed.
   for (int c = 0; c < graphs[NCCL_ALGO_NVLS]->nChannels; ++c) {
     bool addHead = true;
-    int* nvlsIntra = graphs[NCCL_ALGO_NVLS]->intra + c * localRanks;
+    int* nvlsIntra = graphs[NCCL_ALGO_NVLS]->intra + c * nvlsRanks;
```

Comment:
Several places switch to using `nvls*` fields. I know that we've commented out at least one reference to `nvls*` in RCCL. Is this change valid in RCCL?

Reply:
We can ignore any changes related to nvls as it is not supported on AMD GPUs. So just make no change or minimal changes to make further merge easier.